client/resource_group/controller/controller.go

```diff
@@ -344,6 +381,10 @@ func (c *ResourceGroupsController) sendTokenBucketRequests(ctx context.Context,
 		Requests:              requests,
 		TargetRequestPeriodMs: uint64(defaultTargetPeriod / time.Millisecond),
 	}
+	if c.responseDeadlineCh == nil {
+		c.run.responseDeadline.Reset(time.Second)
```

Comment:
How about considering making the deadline timeout be configurable?

Reply:
Good point